src/components/TodoList.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+
+import { Todo } from './Todo';
+import { TodoWithUser } from '../utils/types';
+
+interface Props {
+  todos: TodoWithUser[];
+  onNameButton(): void;
+  onTitleButton(): void;
+  onConditionButton(): void;
+  onTaskButton(): void;
+}
+
+export const TodoList: FC<Props> = (props) => {
+  const {
+    todos,
+    onNameButton,
+    onTitleButton,
+    onConditionButton,
+    onTaskButton,
+  } = props;
+
+  return (
+    <table className='todo__table'>
+      <thead>
+        <tr>
+          <th className='table__title'>
+            <button
+              type="button"
+              onClick={onTaskButton}
```

Comment:
зачем тебе много обработчиков?
сделай один и передавай field
```suggestion
              onClick={() => handleSort('title')}
```